src/util/allIcons.ts

```diff
@@ -0,0 +1,4 @@
+export const ALL_ICONS: Array<string> = require
```

Comment:
ideally we could do some typescript magic here and generate a string union :') but it can't infer the result of this function operation. will investigate in the future